packs/infrastructure/config.yml

```diff
@@ -8,7 +8,43 @@ title: Infrastructure
 
 # Description of the pack
 description: |
-  The infrastructure pack allows you to get visibilility into the performance and availability of your hosts and operating system.
+  ## Infrastructure Monitoring Quickstart
+
+  Infrastructure monitoring provides deep visibility into performance, available resources, and your entire estate at a glance. Remote infrastructure monitoring tools collect and display data and related attributes represented by various metrics and metadata.
+
+  Monitoring helps cloud-based DevOps teams quickly identify and resolve issues, averting potential downtime. By triggering alerts, server monitoring also improves overall flexibility and scalability to manage peaks. You can drill down into specific errors and resolve them to ensure that your environment is fully optimized.
+ 
+  New Relic's infrastructure quickstart provides visibility across the entire stack and enables greater scale and efficiency. It's a monitoring solution for multifaceted hybrid environments with customized dashboards and visualizations.
+
+
+  ### What's Included?
```

Comment:
Having what's included twice seems odd to me. The tiles above the description tells you what's included, so i'd suggest we don't use this pattern or if it's important call it something else.

Reply:
<img width="1498" alt="Screen Shot 2021-09-23 at 12 45 02 PM" src="https://user-images.githubusercontent.com/1395158/134565884-eb1df056-aa87-400f-be3f-cebc088c0887.png">

Reply:
I think we should include it, as the dashboard is much harder to "skim" than the text here is. This gives the user the specifics of what is included in the pack.

I do agree that having "What's included" repeated twice is bad practice though. I agree we should change this instance , I'll push up a commit with a change.

Reply:
I've pushed up a change resolving this.